common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -2,6 +2,7 @@
     "env": {
         "browser": true,
         "es6": true,
+        "jest/globals": true,
         "node": true
```

Comment:
do these mean that these types/objects are assumed to be available when using them?

Reply:
Good point. Ideally we'd have individual configs for every package with the minimal env for each... but I also think that we'll need other mechanisms to help enforce browser-compatible packages, so maybe this is ok.